### PR TITLE
Several API changes

### PR DESCRIPTION
**Pull request by gshaibi**
- [x] All [tests](https://github.com/jfrog/jfrog-client-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-client-go/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

This PR demonstrates the following changes.

AccessManager changes:
- Add RevokeAccessToken method
- Add GetAccessTokens method
- Add TokenId and Username to CreateAccessToken request and response

Artifactory SecurityService changes:
- Add description field to CreateTokenParams

### CODE REVIEW

**Comment by gshaibi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Let's also add tests for the new `RevokeAccessToken` and `GetAccessTokens` APIs added.
You cam use a mock server similar to how its down in the `TestUploadFilesWithFailure` function.

**Comment by gshaibi** [comment]
Hi @eyalbe4 , thanks for the review!
Unfortunately, I don't have much time and I'm not sure I'll be able to fix all the issues you mentioned, but still I'll answer your questions.

File: artifactory/manager.go [hunk 1]
```diff
@@ -393,6 +393,7 @@ func (sm *ArtifactoryServicesManagerImp) GetAPIKey() (string, error) {
 func (sm *ArtifactoryServicesManagerImp) CreateToken(params services.CreateTokenParams) (auth.CreateTokenResponseData, error) {
 	securityService := services.NewSecurityService(sm.client)
 	securityService.ArtDetails = sm.config.GetServiceDetails()
+	// Deprecated
```
[the remark this answers is not in the document]
Comment (gshaibi) [reply artifactory/manager.go:396]: From what I saw on JFrog's docs, this API is deprecated

**Comment by eyalbe4** [comment]
@gshaibi,
The most important piece for getting this PR merged is the missing tests. If you aren't able to continue working on this currently, let's close this PR for now and if possible in the future, resume the work.

**Pull request closed without merging**